deis-tests/manifests/deis-tests-pod.yaml

```diff
@@ -9,17 +9,13 @@ spec:
   restartPolicy: Never
   containers:
   - name: deis-e2e
-    image: quay.io/deisci/deis-e2e:canary
+    image: quay.io/deisci/deis-e2e:git-7f7d775
```

Comment:
i thought it used canary too. but at some point over the last day or 2 we moved to using immutable images in the chart.